### PR TITLE
GL-537: New design for commodity page

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/GL-537

### What?
Design improvements on commodity page

### I have added/removed/altered:
-Move the tree about the commodity info table
-Remove the country row from the table
-Add the status tag “You are currently using the UK(or NI) Online Tariff” + text block beneath it -Update label for the country selection.

### Why?
To improve UI/XI of the commodity page

### Have you? (optional)

- [x] Reviewed view changes with stakeholders
- [x] Validated mobile responsive behaviour of view changes

### Deployment risks (optional)
Very low

### CODE REVIEW

File: spec/views/context_tables/commodity.html.erb_spec.rb
Comment (alexdesi) [inline spec/views/context_tables/commodity.html.erb_spec.rb:59]: The country has been removed from the commodity context table

File: app/webpacker/src/stylesheets/_commodity-codes.scss
Comment (alexdesi) [inline app/webpacker/src/stylesheets/_commodity-codes.scss:66]: This is to avoid the tag to stay in just one line (without it, the tag renders in two lines)

File: config/locales/en.yml
Comment (alexdesi) [inline config/locales/en.yml:98]: an update of the label for the country_picker, required by UCD

**Review by shjohnson — APPROVED** [review]
LGTM once gets green light from UCD merge away! 🍏

**Pull request merged**
